.github/workflows/tests.yml

```
@@ -48,3 +48,11 @@
         PYTHONUNBUFFERED=1;FLASK_ENV=development;FLASK_APP="goodmap.goodmap:create_app(config_path='./tests/e2e_tests/e2e_test_config.yml')" poetry run flask run &
         sleep 3
         cd tests/e2e_tests && node_modules/cypress/bin/cypress run --browser chromium
+    - name: Generate e2e stress test data
+      run: |
+        python tests/e2e_tests/cypress/support/generate_stress_test_data.py
+    - name: Run e2e stress tests
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:1:1: PYTHONUNBUFFERED appears unused. Verify use (or export if used externally)

(shellcheck)